packages/performance-tests/test/cases/prepare-fork.ts

```diff
@@ -0,0 +1,92 @@
+import * as path from "path";
+import * as fs from "fs";
+import * as os from "os";
+import { BriefcaseDb, ExternalSource, ExternalSourceIsInRepository, IModelDb, RepositoryLink, StandaloneDb } from "@itwin/core-backend";
+import { BriefcaseIdValue, Code } from "@itwin/core-common";
+import { IModelTransformer } from "@itwin/imodel-transformer";
+import { initOutputFile, timed } from "../TestUtils";
+import { Logger, StopWatch } from "@itwin/core-bentley";
+import { setToStandalone } from "../iModelUtils";
+import { BriefcaseArgs, ReporterEntry } from "../TestContext";
+
+const loggerCategory = "Transformer Performance Tests Prepare Fork";
+const outputDir = path.join(__dirname, ".output");
+
+export default async function prepareFork(sourceDb: BriefcaseDb, sourceBriefcaseArgs: BriefcaseArgs){
+  let reporterData: ReporterEntry;
+
+  if(!sourceDb.isOpen)
+    sourceDb = await BriefcaseDb.open({
+      fileName: sourceBriefcaseArgs.fileName,
+      readonly: sourceBriefcaseArgs.briefcaseId ? sourceBriefcaseArgs.briefcaseId === BriefcaseIdValue.Unassigned : false,
+    });
+
+  // create a duplicate of master for branch
+  const branchPath = initOutputFile(`PrepareFork-branch.bim`, outputDir);
+  if (fs.existsSync(branchPath))
+    fs.unlinkSync(branchPath);
+  const filePath = sourceDb.pathName;
+  fs.copyFileSync(filePath, branchPath);
+  setToStandalone(branchPath);
+  const branchDb = StandaloneDb.openFile(branchPath);
+
+  let entityProcessingTimer: StopWatch | undefined;
+  try {
+    [entityProcessingTimer] = await timed(async () => {
+      // create an external source and owning repository link to use as our *Target Scope Element* for future synchronizations
+      const masterLinkRepoId = new RepositoryLink({
```

Comment:
move most of this callback into its own function, maybe called `classicalTransformerBranchInit`

Reply:
which part should be put into a function, everything inside of the timer? Otherwise, I don't know how to put the setup in a function when we are currently timing some of that process.

Reply:
yes basically all of it

Reply:
leaving out the duplicating part that we weren't timing, but if you want that included I will add. Otherwise done

Reply:
It looks like moving this to a helper function caused an issue where it seems like it is skipping it. The run time is only like 0.03 seconds. Unresolving comment for now

Reply:
fixed